controlpanel/api/helm.py

```diff
@@ -75,6 +75,13 @@ def upgrade_release(self, release, chart, *args):
             "upgrade", "--install", "--wait", "--force", release, chart, *args,
         )
 
+    def reset_home(self, user):
+        HelmRepository.update()
+
+        return self.__class__.execute(
+            "reset", "--install", "--wait", "--force", user
```

Comment:
I'm not sure I follow this.

AFAICT helm doesn't have a `reset` command, does it? (Checking...) https://helm.sh/docs/helm/

Maybe you meant install an helm chart called `reset` into the user home?

But this seem to try and run `helm reset --install --wait --force USER` which I'm pretty sure would raise an exception as helm doesn't recognise it.

It may be worth trying it.

May I suggest you re-use the `Helm.upgrade_release()` function instead? I just started to have a look at this so I may be wrong but AFAICT you're trying to install a helm release into the user home which contains a `Job` of some sort.

Also, it may be worth specifying the user namespace as well so that this releases would be inside the user namespace with the other user-specific things.

Reply:
This is a placeholder for whatever the real command for helm will be. ;-) (I should have commented this was the case, but it is, after all, a work in progress).